source/PlayState.hx

```diff
@@ -1,33 +1,44 @@
 package;
 
+import js.lib.webassembly.Global;
```

Comment:
I don't think this is used anywhere, and including it prevents the game from building to the neko target via `lime test neko`

I'd like this removed before merging, please.